gpytorch/lazy/root_lazy_tensor.py

```diff
@@ -66,6 +66,9 @@ def _t_matmul(self, rhs):
         # Matrix is symmetric
         return self._matmul(rhs)
 
+    def add_low_rank(self, low_rank_mat, root_decomp_method=None, root_inv_decomp_method="pinverse"):
+        return super().add_low_rank(low_rank_mat, root_inv_decomp_method=root_inv_decomp_method)
```

Comment:
This shouldn't be necessary, `method` of `root_decomposition` will already be ignored in `RootLazyTensor`

Reply:
This is done more to pass in the "pinverse" method for the root inverse decomposition, and to allow that to change if possible.